ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -215,8 +215,12 @@ objects:
 
         # Create the network and firewall rules to attach it to VM
         gcloud compute networks create "${INSTANCE_PREFIX}" \
-          --subnet-mode=auto \
+          --subnet-mode=custom \
           --bgp-routing-mode=regional
+        gcloud compute networks subnets create "${INSTANCE_PREFIX}" \
+          --network "${INSTANCE_PREFIX}" \
+          --range=10.0.0.0/9 \
+          --zone "${GOOGLE_COMPUTE_ZONE}"
```

Comment:
@sallyom Do we really need `--zone` since in the config we are setting it along with project ID?

Reply:
nope that was wrong, we do need region though, i added that/noted your PR